README.md

```diff
@@ -192,3 +191,58 @@ For Unit Testing CppUtest framework is used.
   * Set **BUILD_UNIT_TESTS** option *ON*
   * Set **PATH_CPP_UTEST** Path to the cpputest folder
   * Build and Run the *facemask-plugin-test* project, it will show the report of tests
+  
+  
+  
+## Video Comparison side by side
+
+### Setting up:
+* Install FFmpeg:
+  [ffmpeg](https://www.ffmpeg.org/)
+  **Add a Path** of FFmpeg executable in the Environment Variables:
+  The path should look like: *%PATH_TO_YOUR_DIR%\ffmpeg-%VERSION%\bin*
+  
+### Format Video:
+Formatting video makes start/end of the looped video detectable by adding a red frame at the start
+- In the *facemask-plugin/data* there is a script: **format.bat**
+- Put your sample video as input.mp4 and click on the format.bat
+- Or call the script from the shell 
+```console 
+./format.bat /path/to/the/input/video /path/to/the/output/video 
+```
+Example:
+```console 
+./format.bat inputfolder/myinput.mp4 outputfolder/myoutput.mp4 
+```
+It will generate output with red frame appended
+
+
+### Record Video:
+* Start/Restart Streamlabs-OBS
+* Add a Media Resource
+    * Ensure that **Loop** option is *ON* and **Hide source when playback ends** is *OFF*
+    * Choose video by changing *properties->local file*
+##### There are 2 ways to record video:
+- ##### Manually
+  - Start Steramlabs-OBS recorder *(don't forget to change output format (in settings/output) to mp4)*
+  - Start and End recording as you see **red frame** appears
+- ##### Automotive
+  * Add a filter: Face Mask Plugin
+  * Go to Setting of facemask-plugin:
+    * Set the **Demo Mode Folder** Path (where the output video should be generated)
+	* Turn On **Record Video**
+	* Recording and Rendering will take some time
+	
```

Comment:
When you generate the frames, could you put them in a tmp folder inside of the Demo Folder and then delete tmp folder in the end the way you delete the frames right now? Also, can you name the videos based on the date/time like the ones that are created when recording using REC button